jupyterhub/notebook-images/overlays/additional/generic-data-science-notebook-imagestream.yaml

```diff
@@ -15,11 +15,11 @@ spec:
   tags:
   - annotations:
       opendatahub.io/notebook-software: '[{"name":"Python","version":"v3.8.6"}]'
-      opendatahub.io/notebook-python-dependencies: '[{"name":"Boto3","version":"1.17.11"},{"name":"Kafka-Python","version":"2.0.2"},{"name":"Matplotlib","version":"3.4.2"},{"name":"Numpy","version":"1.21.0"},{"name":"Pandas","version":"1.2.5"},{"name":"Scikit-learn","version":"0.24.2"},{"name":"Scipy","version":"1.7.0"}]'
-      openshift.io/imported-from: quay.io/thoth-station/s2i-generic-data-science-notebook
+      opendatahub.io/notebook-python-dependencies: '[{"name":"Boto3","version":"1.17.11"},{"name":"Kafka-Python","version":"2.0.2"},{"name":"Matplotlib","version":"3.4.1"},{"name":"Numpy","version":"1.19.2"},{"name":"Pandas","version":"1.2.4"},{"name":"Scikit-learn","version":"0.24.1"},{"name":"Scipy","version":"1.6.2"}]'
+      openshift.io/imported-from: quay.io/modh/odh-generic-data-science-notebook
     from:
       kind: DockerImage
-      name: quay.io/thoth-station/s2i-generic-data-science-notebook:v0.0.5
-    name: "v0.0.5"
+      name: quay.io/modh/odh-generic-data-science-notebook:nb-1
+    name: "v1.2.0-2"
```

Comment:
oops, that's supoosed to be "rhods-version" (pushed the result of an experiment)